resources/prometheus/remote-write.yaml

```diff
@@ -1,7 +1,7 @@
 writeRelabelConfigs:
   # Drop useless labels
   - action: labeldrop
-    regex: endpoint|instance|job|prometheus|pod
+    regex: endpoint|instance|job|prometheus
```

Comment:
Isn't there a way to allow the `pod` label only for the system metrics?

Reply:
I am trying to figure out this how this can be resolved. I guess this can be using the action:keep .

Reply:
I have tried to set a  label "pod" for only mentiond metrics and tried it in the local and it is working fine in my local